tools/sync_test_server/Dockerfile

```diff
@@ -5,14 +5,16 @@ ARG ROS_DE_VERSION
 # Add realm repo
 RUN apt-get update -qq \
     && apt-get install -y curl npm \
-    && curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh | bash
+    # && curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh \
```

Comment:
It is mostly an easy reference to the location of the proper releases. I would probably prefer to keep it there for the time being while we are still working on RC candidates.